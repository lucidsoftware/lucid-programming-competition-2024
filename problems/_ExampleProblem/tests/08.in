version https://git-lfs.github.com/spec/v1
oid sha256:012ac0d98a6f21deaccab9a0dc8611e6d047019f2f079551c596d81ad01b21bb
size 41795
