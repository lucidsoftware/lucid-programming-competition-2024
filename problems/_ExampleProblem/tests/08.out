1 rnlbjtfsoz
2 n/a
3 fojatbdrxp gqzyvfdpjr
4 rixekbsnvg
5 wfdopvkylg xiqckdwvay ysfmdqkzic
6 obptknfdva xqpkmboncz
7 n/a
8 n/a
9 qiljovmdwn
10 tgsifowczx
11 jzmhxobtyw nwlsvrmjte
12 pujdsmztvx
13 n/a
14 dpoyuwbjrl kjwncodpyr vkqmoylnsi
15 xikmnlqchd
16 n/a
17 n/a
18 bkieojtwdp pwrmexhlkf xghzqmbpot
19 kqjybzvlpg
20 n/a
21 hsrmowpckq
22 lytkivdpoh
23 gskfwjmtyi pwcjnyvfdl zfiqavlhmx zlyirvpkqe
24 gudtiawzmo
25 rwxcloqfup yvfklzrgax
26 ebcvhrxsuf
27 zlvbdhtyra
28 n/a
29 n/a
30 dbzhtxrwma jxwzthbngr
31 n/a
32 gnsoawqykb
33 bfstxudwop eamjxrkwln stldwmfihg
34 n/a
35 mvqyxowgzi
36 n/a
37 qkmndpzcxi
38 lqtspjzgny oxplvdqugr
39 n/a
40 oiavxrnecj
41 tgzmlwqpjo
42 akcxhrnvme huvkpqszmd
43 n/a
44 izycqkvnhe nyutrdjsmw
45 n/a
46 twsudnjrpm
47 mcwsxgzkjb
48 oamrxcjzlt
49 ytixgcznqo
50 n/a
51 n/a
52 n/a
53 vmpauzbyoj vunrqmjowc
54 cyznwfdjoh
55 n/a
56 xonqzfljdw
57 n/a
58 n/a
59 n/a
60 gmwvlkpyox nrpxyecvjs
61 vkqysfrdwe
62 hkzlfvwrqi pjxnazldfc rvofzjeqlt
63 gcjxylowud
64 mldsqzvgoh utvlnarzox
65 n/a
66 krfzcmqlae
67 jvhrlfsyew
68 sbzrtogfap vnhqjptulb
69 n/a
70 gfhkwcyxvi
71 n/a
72 hqculzsjvx wmzxsugrye
73 cikxuwdmho epfyskbnjh
74 jexkczbuvp kwifucjzpg
75 fdvnmbkjro
76 n/a
77 jfrilzqyhx lxzqoyprud
78 hjckflsqie
79 qvruyfnszc
80 n/a
81 n/a
82 n/a
83 n/a
84 n/a
85 n/a
86 pzrhagxsyk
87 adiwvsqumx
88 n/a
89 wlmohrzpkv
90 ctowvqkpry
91 n/a
92 fbyjvlwrxs yqhdunpilg
93 zlgfijmrnx
94 bqsoewjygc cewnzflmoj udtxyacmwp
95 nmewygfdtz
96 nqveikjwrx
97 rqefdxjvbn
98 yefctoxpzu
99 rzjfcvwlha
100 qxdvumeyrb
101 luhozsjkwq xjobqpktzl
